src/vea/tests/vea_ut.c

```diff
@@ -227,14 +228,14 @@ ut_reserve(void **state)
 
 	/* Verify transient is allocated */
 	rc = vea_verify_alloc(args->vua_vsi, true, off_a, blk_cnt);
-	assert_int_equal(rc, 0);
+	assert_rc_equal(rc, 0);
 	/* Verify persistent is not allocated */
 	rc = vea_verify_alloc(args->vua_vsi, false, off_a, blk_cnt);
-	assert_int_equal(rc, 1);
+	assert_rc_equal(rc, 1);
```

Comment:
the ones that expect 1 are a little weird.  At the least, they need your patch to update the macro.

Reply:
Agreed, this is weird. And in certain cases (rc doesn't map to a DER value) the assert comparing the strings in the macro just won't detect the failure.